### PR TITLE
LPS-139983 Disable the publish button until the save action is completed

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issue: https://issues.liferay.com/browse/LPS-139983

Hello Reviewer! 🌸

This PR has a fix related to the bug linked above. 
If you have any questions feel free to ask, thank you for your time!

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by miltonmc — CHANGES_REQUESTED** [review]
Just a small tweak:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js [hunk 1]
```diff
@@ -312,6 +312,12 @@ export const FormBuilder = () => {
 		(event) => {
 			event.preventDefault();
 
+			const publishButton = document.querySelector(
+				'.lfr-ddm-publish-button'
+			);
+
+			publishButton.classList.add('disabled');
```
[the remark this answers is not in the document]
Comment (rebsilva) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js:319]: Thanks Milton, I will try this!

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by miltonmc — APPROVED** [review]

**Comment by liferay-forms** [comment]
Hey @rebsilva, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**